app/controllers/entry_controller.rb

```diff
@@ -4,16 +4,14 @@ class EntryController < ApplicationController
   before_filter :load_entry, :only => [:comment]
 
   def comment
-    begin
-      if @challenge.participator?(current_user) && params[:comment] && params[:comment][:text].size > 0
-        @entry.comments.push Comment.new(:comment => params[:comment][:text], :nickname => current_user.nickname)
-        @challenge.save
-      end
-
-    rescue Exception => e
-    ensure
-      redirect_to challenge_path(params[:challenge])
+    if @challenge.participator?(current_user) && params[:comment] && params[:comment][:text].size > 0
+      @entry.comments.push Comment.new(:comment => params[:comment][:text], :nickname => current_user.nickname)
+      @challenge.save
     end
+  rescue
```

Comment:
this feels a little odd. could `@entry` possibly be `nil` is that why this is here? not a huge deal but curious what your thoughts on adding the redirect here as well? it's usually bad practice to have empty `rescue` blocks